buildchain/buildchain/targets/local_image.py

```diff
@@ -162,9 +162,20 @@ def check_dockerfile_dependencies(self) -> Union[TaskError, bool]:
             return TaskError(msg=error_message)
         return None
 
+    def clean_registry(self) -> Optional[TaskError]:
```

Comment:
minor: you're not really cleaning the registry, `rm_from_registry` would be a slightly better name IMHO